### PR TITLE
[Synapse] Add synapse Trigger Run commands

### PR DESCRIPTION
**Pull request by kevinzz6**
## Description

Add some cmdlets to manage Trigger run in synapse workspace

- Add new cmdlet to invokes another instance of a trigger run.
- Add new cmdlet to stops a trigger run.
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1017
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Synapse/Synapse/Properties/Resources.resx [hunk 1]
```diff
@@ -693,6 +693,11 @@ Are you sure you want to continue?</value>
   <data name="WorkspaceGitRepoParameterException" xml:space="preserve">
     <value>'{0}' is not provided.</value>
   </data>
+  <data name="StoppingSynapseTriggerRun" xml:space="preserve">
+    <value>Stopping trigger run with Id: '{0}'</value>
+  </data>
+  <data name="RerunSynapseTriggerRun" xml:space="preserve">
+    <value>Rerunning trigger run with Id: '{0}'</value>
```
Comment (idear1203) [inline src/Synapse/Synapse/Properties/Resources.resx:700]: Add ` </data>` after this line.

File: src/Synapse/Synapse/help/Invoke-AzSynapseTriggerRun.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Invoke-AzSynapseTriggerRun -WorkspaceName ContosoWorkspace -Name ContosoTrigger -TriggerRunId 000111222333abc
+```
+
+Starts another instance of a trigger run with a new trigger run id, keeping the same windowStartTime and windowEndTime as the original trigger run.
+
+## Example 2
```
Comment (idear1203) [inline src/Synapse/Synapse/help/Invoke-AzSynapseTriggerRun.md:45]: ### Example 2

File: src/Synapse/Synapse/help/Stop-AzSynapseTriggerRun.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PassThru
+{{ Fill PassThru Description }}
```
Comment (wyunchi-ms) [inline src/Synapse/Synapse/help/Stop-AzSynapseTriggerRun.md:109]: Please replace the placeholder.

File: src/Synapse/Synapse/help/Invoke-AzSynapseTriggerRun.md [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PassThru
+{{ Fill PassThru Description }}
```
Comment (wyunchi-ms) [inline src/Synapse/Synapse/help/Invoke-AzSynapseTriggerRun.md:109]: Please replace the placeholder.

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**